docs/api.md

```diff
@@ -555,7 +555,7 @@ File name, as specified by the torrent. *Example: 'some-filename.txt'*
 
 ## `file.path`
 
-File path, as specified by the torrent. *Example: 'some-folder/some-filename.txt'*
+File path, including torrent path. *Example: 'my-torrent/some-folder/some-filename.txt'*
```

Comment:
Can you expand on this?  
What is wormhole doing where this would break?

The file.path really doesn't seem to be a safe indicator of a file location, as the torrent and files are backed by chunk-stores.  In a we (and I'd argue even in a native install) getting a file stream through the apis is a safer way to refer to files than getting the file object and trying to use the path.

To me, this information seems more for display purposes than actually useful.  The store is the one who should decide where the file actually resides.

Reply:
Is the wormhole code open source?   Can I see how file.path is being used there?